src/admin/content-block/components/ContentBlockPreview/ContentBlockPreview.tsx

```diff
@@ -26,6 +26,11 @@ interface ContentBlockPreviewProps {
 	blockState: ContentBlockState;
 }
 
+const CONTENT_WIDTH_MAP: { [key in ContentWidth]: 'regular' | 'large' | 'medium' } = {
+	REGULAR: 'regular',
+	LARGE: 'large',
+	MEDIUM: 'medium',
```

Comment:
zijn deze dan ook niet beter capitalized?

Reply:
Nee, die map is bedoeld voor de Container component omdat zijn size prop lowercase is